src_kt/javatests/arcs/crdt/CrdtCountTest.kt

```diff
@@ -0,0 +1,149 @@
+package arcs.crdt
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [CrdtCount]. */
+@RunWith(JUnit4::class)
+class CrdtCountTest {
+  lateinit var alice: CrdtCount
+  lateinit var bob: CrdtCount
+
+  @Before
+  fun setup() {
+    alice = CrdtCount()
+    bob = CrdtCount()
+  }
+
+  @Test
+  fun count_initializes_to_zero() {
```

Comment:
Shouldn't these be camel cased? `countInitializesToZero`

https://developer.android.com/kotlin/style-guide#function_names

Reply:
Tests are typically okay to have some snake-case going on, but you're right - underscores should only really be used as a 'comma' when reading the test name. Thanks!